packages/auth/src/utils/send-mail.ts

```diff
@@ -0,0 +1,22 @@
+import { BACKEND_URL } from "./constants";
+
+export async function sendMail({ to, subject, text }: { to: string; subject: string; text: string }) {
+	try {
+		const response = await fetch(`${BACKEND_URL}/api/email/send-mail`, {
+			method: "POST",
+			headers: {
+				"Content-Type": "application/json",
+			},
+			body: JSON.stringify({ to, subject, text }),
+		});
+
+		if (!response.ok) {
+			throw new Error(`Failed to send email: ${response.statusText}`);
+		}
+
+		return await response.json();
+	} catch (error) {
+		console.error("Error sending email:", error);
+		throw error;
+	}
+}
```

Comment:
```suggestion
import { BACKEND_URL } from "./constants";
import axios from "axios";

export async function sendMail({ to, subject, text }: { to: string; subject: string; text: string }) {
    try {
        const response = await axios.post(
            `${BACKEND_URL}/api/email/send-mail`,
            { to, subject, text }
        );

        return response.data;
    } catch (error) {
        console.error("Error sending email:", error);

        throw error;
    }
}
```

Why not use axios like this?